tooling/nargo_cli/src/cli/debug_cmd.rs

```diff
@@ -120,7 +120,7 @@ pub(crate) fn debug_program(
 
     noir_debugger::debug_circuit(
         &blackbox_solver,
-        compiled_program.circuit.clone(),
+        &compiled_program.circuit.clone(),
```

Comment:
```suggestion
        &compiled_program.circuit,
```

We don't need to clone here as we're passing by reference.